examples/nextjs-app/src/styles/StyledComponentsRegistry.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { useServerInsertedHTML } from "next/navigation";
+import React, { useState } from "react";
+import { ServerStyleSheet, StyleSheetManager } from "styled-components";
+
+export function StyledComponentsRegistry({
```

Comment:
taken from https://nextjs.org/docs/app/building-your-application/styling/css-in-js#styled-components